src/devtools/client/debugger/src/components/Editor/Breakpoints/Breakpoints.css

```diff
@@ -36,6 +36,14 @@
   pointer-events: all;
 }
 
+.empty-line .line-action-button {
+  pointer-events: none !important;
+}
+
+.empty-line .toggle-widget {
+  background: #8abfd8 !important;
+}
```

Comment:
Eeek 😬 specificity selectors _and_ `!important`

Reply:
oh boy - what have we done to deserve this :P